### ISSUE TITLE
Plan Sums?

### ISSUE THREAD

**Issue by justinengl** [Varian-Innovation-Center/MAAS-PlanScoreCard#8]
Looks like there is not a way to do Plan Sums in this. Is this correct? Thanks!

**Comment by mattcschmidt** [comment]
Yes, your assumption is correct. The decision early on was to generate scorecards for each component of the plansum, rather than for the plansum as a whole.

**Comment by justinengl** [comment]
Great. Thanks for quick response!
